grpc-health-probe.yaml

```diff
@@ -23,12 +23,9 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 1089f40639043ee46cc47cbea1bcb0cc48cfc284
 
-  - uses: go/build
+  - uses: goreleaser/build
     with:
-      packages: .
-      output: grpc_health_probe
-      tags: netgo
-      ldflags: -s -w
+      args: --snapshot
```

Comment:
do we need `--snapshot` or can we use the release?  I think this will change the version displayed by the `--version` and we'd want to indicate these are real releases rather than development ones?

Reply:
snapshot option is just for avoiding to release these artifacts in GitHub releases

Reply:
there was an option for snapshot builds, I've updated the build to use that.

Reply:
though using `snapshot` will still mean the version of the binary will include a `-devel` suffix?  I wonder if there's a way to avoid publishing the artifacts and still use a release version?

Reply:
let me try with that, thanks James!

Reply:
without snapshot option the pipeline is failing due to git dirty state

Reply:
I used --skip-validate to avoid having git dirt state issues, does that make sense?